### DISCUSSION TITLE
LCD XPT2046

### DISCUSSION THREAD

**Discussion by cdevalland** [rern/rAudio#781] [Feature Requests]
Hi,
thanks for your rAudio-1 !
I have a XPT2046 LCD Display. [https://github.com/goodtft/LCD-show/blob/master/LCD35-show](url)
Do you know how to enable this display in rAudio ?
I tried generic and waveshare devices but it didn't work.
Regards

**Comment by rern** [comment]
Mine is a generic XPT2046 as well and it's doing fine.
Don't forget to reboot after enabled.

**Reply by cdevalland** [reply]
I tried GENERIC and reboot but screen turns to black and nothing displays :(

**Reply by rern** [reply]
The screen turning to black is a good sign. Disable `Browser on RPi` should bring terminal (text mode) to display.

Try:
- Get a latest image file
- After 1st boot, Addons > rAudio 1 > Update
- System > enable TFT 3.5" LCD
- Reboot
- The LCD should display "Page Unresponsive" > `Exit page` > "Aw, Snap!" > `Reload`

To fix "Aw, Snap!":
- Addons > Chromium downgrade

**Reply by cdevalland** [reply]
Thanks but still black screen after reboot...

**Reply by cdevalland** [reply]
It's OK with latest release of rAudio 1.
Thanks !

**Comment by cdevalland** [comment]
It's OK with last update.
